v1/translate.yaml

```diff
@@ -0,0 +1,254 @@
+info:
+  version: '1.0.0'
+  title: Content Translation API
+  description: Content Translation API
+  termsofservice: https://github.com/wikimedia/restbase
+  license:
+    name: Apache licence, v2
+    url: https://www.apache.org/licenses/LICENSE-2.0
+paths:
+  /machine/{from}/{to}{/provider}:
```

Comment:
It doesn't really matter whether it is html or plain-text, we can translate both. In fact the fact that we always wrap the result unconditionally in a div is an anti-feature: https://phabricator.wikimedia.org/T87656

But calling it `html` would allow adding complemeting `text` entry point that does not do the wrapping, thus solving the reported issue, so I too like this suggestion.